rosette/rbl/rosette/configuration.rbl

```diff
@@ -1,47 +1,49 @@
 ;;; Mode: -*- RBL -*-
 ;;; @BC
-;;;		                Copyright (c) 1993
-;;;	    by Microelectronics and Computer Technology Corporation (MCC)
-;;;				All Rights Reserved
+;;;                             Copyright (c) 1993
```

Comment:
I don't know if this is a related change or not. I'd rather not see it.

Reply:
This is part of a commit that is only in this PR.

Reply:
No, you reformatted everything.
Can you not do that so the diff is smaller?

If you want to reformat, do it in a separate PR, that doesn't change anything.
Otherwise it's hard to see what actually changed in this file.

Reply:
Yeah I meant that this wasn't a cherry-picked duplicate from some other PR. I did indeed reformat the whole file. In any case, made the diff smaller.